Nodejs/Product/TestAdapter/TestDiscoverer.cs

```diff
@@ -29,16 +30,16 @@ public TestDiscoverer()
         /// <param name="discoverySink">Callback used to notify client upon discovery of test cases</param>
         public void DiscoverTests(IEnumerable<string> sources, IDiscoveryContext discoveryContext, IMessageLogger logger, ITestCaseDiscoverySink discoverySink)
         {
-            ValidateArg.NotNull(sources, "sources");
-            ValidateArg.NotNull(discoverySink, "discoverySink");
-            ValidateArg.NotNull(logger, "logger");
+            ValidateArg.NotNull(sources, nameof(sources));
+            ValidateArg.NotNull(discoverySink, nameof(discoverySink));
+            ValidateArg.NotNull(logger, nameof(logger));
 
             var env = new Dictionary<string, string>();
-            var root = Environment.GetEnvironmentVariable(NodejsConstants.NodeToolsVsInstallRootEnvironmentVariable);
-            if (string.IsNullOrEmpty(root))
-            {
-                root = Environment.GetEnvironmentVariable("VSINSTALLDIR");
-            }
+            var root = Environment.GetEnvironmentVariable("VSINSTALLDIR");
```

Comment:
I believe the old env var used to be set in case it was being run for another context that VS/Developer Command Prompt. Is that no longer relevant?

Reply:
The old environment variable was set by the package, based on the value of "VSAPPIDDIR" which no longer seems to be set...  the VSInstaldir is consistently set by both VS and the dev cmd